config/rbac/leader_election_role.yaml

```diff
@@ -18,6 +16,10 @@ rules:
   - update
   - patch
   - delete
+  - apiGroups:
```

Comment:
The indentation here does not align with other apiGroups and resources

Reply:
Which leads me to ask how was this tested?

Reply:
Fixed

Reply:
I was going off the indentation in the upgrade guide. When I did a local build it did not catch this. Thanks Mansi.